templates/partials/dropdown.html

```diff
@@ -0,0 +1,392 @@
+                {"share_icon": "<i class='fa-solid fa-square-envelope'></i>"}
+            ],
+
+        },
+        "small_dropdown_list": {
+            "menu_class": "dropdown-small-menu",
+            "menu_title_class": "dropdown-small-title",
+            "container_class":"dropdown-container dropdown-small-container",
+            "label_container_class": "dropdown-small-label-container",
+            "label_class": "dropdown-small-list-label",
+            "chevron_class": "dropdown-small-chevron",
+            "list_class": "dropdown-small-list",
+            "list_item_class": "dropdown-small-item"
+        },
+        "medium_dropdown_list": {
+            "menu_class": "dropdown-medium-menu",
+            "menu_title_class": "dropdown-medium-title",
+            "container_class":"dropdown-container dropdown-medium-container",
+            "label_container_class": "dropdown-medium-label-container",
+            "label_class": "dropdown-medium-list-label",
+            "chevron_class": "dropdown-medium-chevron",
+            "list_class": "dropdown-medium-list",
+            "list_item_class": "dropdown-medium-item"
+        },
+        "dropdown_time_frame": {
+            "container_class": "dropdown-timeframe-container",
+            "content_container_class": "timeframe-content-container",
+            "label_text_class": "button",
+            "timeframe_start_class": "timeframe-start",
+            "timeframe_end_class": "timeframe-end",
+            "datepicker_start_id": "inputDateStart",
+            "timepicker_start_id": "inputTimeStart",
+            "datepicker_end_id": "inputDateEnd",
+            "timepicker_end_id": "inputTimeEnd",
+            "datepicker_end_on_id": "inputDateEndOn",
+            "menu_class": "dropdown-timeframe-menu",
+            "menu_title_class": "subtitle-1"
+        }
+    }
+%}
+
+
+{% if dropdown_type == "default_dropdown_list" %}
+    <div  id="{{ container_id }}" class="{{ dropdown_library[dropdown_type]["container_class"] }}">
+        <label  id={{ label_id }} class="dropdown-default-list-label" for="touchDefaultList">
+            <h6>{{ dropdown_label_text|safe }}</h6>
+            <i id="{{ chevron_id }}" class="fa-solid fa-chevron-down dropdown-chevron"></i></label>
+        <input type="checkbox" id="touchDefaultList">
+
+        <ul class="dropdown-default-list">
+            {% for item in dropdown_library[dropdown_type]["list_items"] %}
+
+                <li><hr class='dropdown-custom-divider'></li>
+                {% with %}
+                    {% set checkbox_label = item["item_label"]|safe %}
+                    {% set checkbox_id = item["item_id"] %}
+                    {% include "partials/checkbox.html" %}
+                {% endwith %}
+            {% endfor %}
+        </ul>
+    </div>
+{% endif %}
+
+
+{% if dropdown_type == "regular_dropdown_list_metadata" or dropdown_type == "regular_dropdown_list_license" or dropdown_type == "regular_dropdown_list_frequency"%}
```

Comment:
Hover colours are not consistent with what is shown in the design document in Figma

Reply:
The colour applied is what is noted in the design:

background: rgba(136, 144, 194, 0.25);

Reply:
As described in library 2:

- if not expanded: grey background
- if expanded and no hover: white background
- if expanded and hover: grey background